### DISCUSSION TITLE
Can't run concurrent jobs on different GPUs

### DISCUSSION THREAD

**Discussion by davidmcnabnz** [basujindal/stable-diffusion#185] [General]
It's great to have the ```--device``` option for specifying particular GPUs to run on. However, if I have one job running on one GPU, and I start another job on a different GPU, it crashes with:

```
Global seed set to nnnnnn
Loading model from models/ldm/stable-diffusion-v1/model.ckpt
Global Step: nnnnnn
UNet: Running in eps-prediction mode
Killed
```
Is there any way around this?

**Comment by trls888s** [comment]
Run them as docker containers using Nvidia-Docker, then you can point the whole container at a GPU.

I have been very successful with this utilizing an old cryptomine to run 4 instances of Stable-Diffusion
